### PR TITLE
feat(new-primary-school): Implement school when to start page

### PR DESCRIPTION
**Pull request by birkirkristmunds**
# ...

[TS-800](https://dit-iceland.atlassian.net/browse/TS-800)

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts [hunk 1]
```diff
@@ -430,7 +431,22 @@ export const NewPrimarySchoolForm: Form = buildForm({
           title:
             newPrimarySchoolMessages.primarySchool
               .startingSchoolSubSectionTitle,
-          children: [],
+
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:434]: Remove new line
```suggestion
```

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts [hunk 2]
```diff
@@ -430,7 +431,22 @@ export const NewPrimarySchoolForm: Form = buildForm({
...
+              id: 'startingSchoolMultiField',
+              title: newPrimarySchoolMessages.primarySchool.startingSchoolTitle,
+              description:
+                newPrimarySchoolMessages.primarySchool
+                  .startingSchoolDescription,
+              children: [
+                buildDateField({
+                  id: 'startDate',
+                  title: newPrimarySchoolMessages.shared.date,
+                }),
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:446]: Missing a placeholder

**Review by veronikasif — COMMENTED** [review]
Shouldn't we add some validation? And maybe disable dates in the past?

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts (hunk 1 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts]: Fixed

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts (hunk 2 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts]: placeholder added

**Comment by birkirkristmunds** [comment]
Added validation and min and max date

**Review by veronikasif — APPROVED** [review]
LGTM 👍

**Pull request merged**